### PR TITLE
Darix-inspired package spec 2899

### PR DESCRIPTION
**Pull request by kirushik**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
@nadjaw is your reviewer :dancers: check https://trello.com/c/AFNeVycP/2899-cnt-switch-from-gem2rpm-to-plain-spec-file-fix-suseconnect-update 
@nadjaw: Please review this pull request using our guidelines: 
* test for acceptance criteria / functionality 
* check if the new code is covered with tests 
* check for unintended consequences 
* encourage usage of the boyscout rule 
* make sure the code is architected in the best way 
* check that no unnecessary technical debt got introduced 
* make sure that no unnecessary FIXMEs or TODOs got added

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
http://river.suse.de/job/scc-connect-prophet/23263/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 30617d49844380a027433679472ab7d29bd1fa10 into 9b6a348a59e37f6c8c863d6901ddeee2b1b3050b )

File: package/SUSEConnect.spec [hunk 1]
```diff
@@ -16,35 +16,37 @@
...
-Source1:       %{name}.5.gz
-Source2:       %{name}.8.gz
-Source3:       %{name}.example
+Source:         %{mod_full_name}.gem
+Source1:        %{name}.5.gz
+Source2:        %{name}.8.gz
+Source3:        %{name}.example
 
-Summary:        SUSE Connect utility to register a system with the SUSE Customer
+Summary:        Utility to register a system with the SUSE Customer Centre
```
Comment (digitaltom) [inline package/SUSEConnect.spec]: SUSE Customer Center

File: package/SUSEConnect.spec [hunk 2]
```diff
@@ -97,11 +84,26 @@ if [ -s /etc/suseRegister.conf ]; then
...
+%{gem_base}/cache/%{mod_full_name}.gem
+%{gem_base}/specifications/%{mod_full_name}.gemspec
+
+%doc %{_mandir}/man5/SUSEConnect.5.gz
+%doc %{_mandir}/man8/SUSEConnect.8.gz
 
+%config(noreplace) %ghost %{_sysconfdir}/SUSEConnect
+%config %{_sysconfdir}/SUSEConnect.example
+%dir %{_sysconfdir}/zypp/
+%dir %{_sysconfdir}/zypp/credentials.d/
```
Comment (digitaltom) [inline package/SUSEConnect.spec]: Do we need to include those directories?

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
http://river.suse.de/job/scc-connect-prophet/24602/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged bf7f457b6e5ffa1720960b7bfb83124ce7f5bda9 into 9b6a348a59e37f6c8c863d6901ddeee2b1b3050b )

File: package/SUSEConnect.spec (hunk 2 above)
Comment (wstephenson) [inline package/SUSEConnect.spec]: Yes, to allow the %ghost on SCCcredentials.

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
http://river.suse.de/job/scc-connect-prophet/24605/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged dd88da90f9d22316e3f3a6dc9ae55a8b9f4b0f0b into 9b6a348a59e37f6c8c863d6901ddeee2b1b3050b )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
http://river.suse.de/job/scc-connect-prophet/24612/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged ffc2211bfa65f13ae30a58f6b9ea848bed90d759 into 9b6a348a59e37f6c8c863d6901ddeee2b1b3050b )

**Comment by NadjaW** [comment]
lgtm

**Pull request merged**